README.md

```diff
@@ -1 +1,254 @@
+        <li><a href="https://www.behance.net/">Behance</a></li>
+      </ul>
+  </details>
+
+### Key Features <a name="key-features"></a>
+
+- Add new Book
+- Remove Book
+- Single page application
+- Dynamic page
+- Redux 
+- Styling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Coming Soon]
+
+## 🚀 Live Recording <a name="live-demo"></a>
+
+[Coming Soon]
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git version 2.38.x
+- node.js version > 12.x
+- IDE
+- browser (chrome, firefox, edge, safari)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Javascript-capston
+
+  git https://github.com/faisalakb/Space-Travelers
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Javascript-capston
+  node npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+run live server
+
+### Run tests
+
+To run tests, run the following command:
+Run Github Actions Test
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx eslint .
+  npm test
+```
+
+### Deployment
+
+This project is deployed at github pages you can clone it here. [Please click to clone](coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors
+
+👤 **Faisal Mumtaz**
+
+- GitHub: [@faisalakb](https://github.com/faisalakb)
+- Twitter: [@faisalakb786](https://twitter.com/Faisalakb786)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/faisal-mumtaz-514a221a6/)
+
+
+
+👤 **Bett Enoch**
+
+- GitHub: [@bettenoch](https://github.com/Bettenoch)
+- Twitter: [@BybEnoch](https://twitter.com/BybEnoch)
+- LinkedIn: [@GeoIntelligence](https://www.linkedin.com/in/bett-kipngeno-enock-8b5153214/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Project 5: Full website.**
+- [ ] **Project 6: Tests (pair programming).**
+  
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/faisalakb/Space-Travelers/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star to this repositiory.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse full-stack curriculum for the inspiration and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is under [MIT](https://github.com/faisalakb/Space-Travelers/blob/setup/LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The link to your LICENSE file is broken and displays a 404 error page. Kindly fix the link so that anyone who intends to use your code understands the legal ramification of doing so. 👍 😎